package-lock.json

```diff
@@ -1175,7 +1175,7 @@
       "handlebars": {
          "version": "4.7.6",
          "resolved": "https://registry.npmjs.org/handlebars/-/handlebars-4.7.6.tgz",
-         "integrity": "sha512-1f2BACcBfiwAfStCKZNrUCgqNZkGsAT7UM3kkYtXuLo0KnaVfjKOyf7PRzB6++aK9STyT1Pd2ZCPe3EGOXleXA==",
+         "integrity": "sha1-1MBcG6+Q6ZRfd6pop6IZqkp9904=",
```

Comment:
I don't think we want that...

Reply:
Don't know why it did that. Reverted in latest commit